src/bt_test/src/bur_nodes.h

```diff
@@ -10,7 +10,12 @@ using std::placeholders::_1;
 namespace BurNodes{
 
 bool gateVisited = false;
+bool buoyVisited = false;
+bool binVisited = false;
 bool torpedoVisited = false;
+bool samplesVisited = false; 
+// Should we have a separate variable to track visiting
+// the pinger in the octagon and then descending to the samples?
```

Comment:
Since this is just a code skeleton, I think we can leave it for now and add another variable later if needed